modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/v2_1_0/util/CPInstanceTable.java

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.product.internal.upgrade.v2_1_0.util;
+
+import java.sql.Types;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author	  Brian Wing Shun Chan
+ * @generated
+ */
+public class CPInstanceTable {
+
+	public static final String TABLE_NAME = "CPInstance";
+
+	public static final Object[][] TABLE_COLUMNS = {
+		{"uuid_", Types.VARCHAR}, {"externalReferenceCode", Types.VARCHAR},
+		{"CPInstanceId", Types.BIGINT}, {"groupId", Types.BIGINT},
+		{"companyId", Types.BIGINT}, {"userId", Types.BIGINT},
+		{"userName", Types.VARCHAR}, {"createDate", Types.TIMESTAMP},
+		{"modifiedDate", Types.TIMESTAMP}, {"CPDefinitionId", Types.BIGINT},
+		{"CPInstanceUuid", Types.VARCHAR}, {"sku", Types.VARCHAR},
+		{"gtin", Types.VARCHAR}, {"manufacturerPartNumber", Types.VARCHAR},
+		{"purchasable", Types.BOOLEAN}, {"width", Types.DOUBLE},
+		{"height", Types.DOUBLE}, {"depth", Types.DOUBLE},
+		{"weight", Types.DOUBLE}, {"price", Types.DECIMAL},
+		{"promoPrice", Types.DECIMAL}, {"cost", Types.DECIMAL},
+		{"published", Types.BOOLEAN}, {"displayDate", Types.TIMESTAMP},
+		{"expirationDate", Types.TIMESTAMP},
+		{"lastPublishDate", Types.TIMESTAMP},
+		{"overrideSubscriptionInfo", Types.BOOLEAN},
+		{"subscriptionEnabled", Types.BOOLEAN},
+		{"subscriptionLength", Types.INTEGER},
+		{"subscriptionType", Types.VARCHAR},
+		{"subscriptionTypeSettings", Types.CLOB},
+		{"maxSubscriptionCycles", Types.BIGINT},
+		{"deliverySubscriptionEnabled", Types.BOOLEAN},
+		{"deliverySubscriptionLength", Types.INTEGER},
+		{"deliverySubscriptionType", Types.VARCHAR},
+		{"deliverySubTypeSettings", Types.VARCHAR},
+		{"deliveryMaxSubscriptionCycles", Types.BIGINT},
+		{"unspsc", Types.VARCHAR}, {"status", Types.INTEGER},
```

Comment:
I think normally we would want this *table.java to be generated with the schema that it was at. Notice how `unspsc` is suppose to be added in v2_6_0 and not here. I think its probably ok though, since its only adding that new field in and not populating it or anything.